plugins/flowdock/app/assets/javascripts/flowdock/application.js

```diff
@@ -1,112 +1,2 @@
 //= require jquery_mentions_input/jquery.elastic.source.js
 //= require jquery_mentions_input/jquery.mentionsInput.js
```

Comment:
Can we load these two guys in `_notify_buddy_box.html.erb` and get rid of this AngularJS file altogether?